client/src/components/Q&A/QuestionContainer.jsx

```diff
@@ -6,17 +6,24 @@ export default function QuestionContainer({ searchTerm, setShow }) {
   const [count, setCount] = useState(2);
 
   function incrementQuestionCount() {
-    setCount(count + 2);
+    setCount((previousCount) => previousCount + 2);
   }
 
   return (
     <>
       <div
         className="questions-container"
       >
-        <QuestionRow searchTerm={searchTerm} count={count} />
+        <QuestionRow
+          searchTerm={searchTerm}
+          count={count}
+        />
       </div>
-      <AddQuestion setShow={setShow} count={count} setCount={incrementQuestionCount} />
+      <AddQuestion
+        setShow={setShow}
+        count={count}
```

Comment:
remove line 23 since this function lives in context now.

Reply:
Indeed, nice catch!